src/main/java/io/getunleash/repository/HttpToggleFetcher.java

```diff
@@ -13,6 +13,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+@Deprecated
```

Comment:
Including a comment explaining what replaces the deprecated API is very helpful. That way the user has a hint of how to migrate.